semgrep_output_v1.atd

```diff
@@ -80,7 +80,7 @@ type position
   (* Byte position from the beginning of the file, starts at 0.
    * OCaml code sets it correctly. Python code sets it to a dummy value (-1).
    *)
-  offset: int;
+  ~offset: int;
```

Comment:
Could be good to put a deprecated notice like (* Semgrep CLI version <= 1.xx.0 was not always producing an offset, hence the ~, but once we don't support that old version in the App, we can remove the ~ *)